src/main/java/seedu/address/logic/parser/ViewTaskCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ViewTaskCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ViewTaskCommand object.
+ */
+public class ViewTaskCommandParser implements Parser<ViewTaskCommand> {
+
+    public static final String MESSAGE_INVALID_NUMBER_OF_INDICES = "Invalid number of indices";
```

Comment:
Would it be better to remove this line, since `MESSAGE_INVALID_NUMBER_OF_INDICES` is not used at all?